### PR TITLE
Add seed trajectories

### PR DESCRIPTION
**Pull request by mlautman**
This takes the comments and commits from https://github.com/ros-planning/moveit_msgs/pull/46

I would open a PR to @bmagyar's branch but then it would likely not get as many eyes on it. @bmagyar feel free to take a portion or all of the changes in  82c504f0e2bf8930355d171110de297436e75a8f and apply them to your branch then close this PR. 

If this is something we can all agree on, then let's get this merged. Seeing as this has gotten plenty of review and the changes only target master, I propose we merge it and make changes later as they come up.

### CODE REVIEW

File: msg/GenericTrajectory.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Header header
```
Comment (BryceStevenWilley) [inline msg/GenericTrajectory.msg]: Header here is redundant, as both `cartesian_trajectory` and `joint_trajectory` have one.

File: msg/GenericTrajectory.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+trajectory_msgs/JointTrajectory joint_trajectory
```
Comment (BryceStevenWilley) [inline msg/GenericTrajectory.msg]: I agree with @v4hn's comment [here](https://github.com/ros-planning/moveit_msgs/pull/46/files#r313735106), there probably doesn't need to be an explicit field choosing which trajectory, but I'd at least add a comment saying that the unintended trajectory will be empty.

File: msg/CartesianPoint.msg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+geometry_msgs/Pose pose
+geometry_msgs/Twist velocity
+geometry_msgs/Accel acceleration
```
Comment (BryceStevenWilley) [inline msg/CartesianPoint.msg]: Nit: need newline here?

Comment (bmagyar) [reply msg/CartesianPoint.msg]: I don't think there's a consistent standard for this, at least not in this package :)

**Comment by bmagyar** [comment]
Thanks @mlautman . I have some ideas for the stale comments on the other MR, I've taken all your changes over there and closing this as advised.

**Pull request closed without merging**